docs/Tutorial%3A-How-to-Configure-the-SMART-on-FHIR-Sample-Application-using-OSS-FHIR-Server.md

```diff
@@ -0,0 +1,55 @@
+You can follow the steps outlined in the document, "[Tutorial: Azure Active Directory SMART on FHIR proxy](https://docs.microsoft.com/en-us/azure/healthcare-apis/use-smart-on-fhir-proxy)." However, there are a few differences, as documented here.
+
+# Enable the SMART on FHIR proxy
+
+Make sure that the values of FhirServer:Security:EnableAadSmartOnFhirProxy and FhirServer:Security:Enabled are set to True.
+
+![image.png](/docs/images/SMARTonFHIR/image-adb646f3-ad32-4ce4-8a73-9376ca82f1a2.png)
+
+The OSS FHIR Server security is handled through the server (or app) using four custom defined [roles](https://github.com/microsoft/fhir-server/blob/master/src/Microsoft.Health.Fhir.Shared.Web/roles.json), globalAdmin, globalWriter, globalReader and globalExporter. It is not done through the portal, and therefore you can leave the App Service Automation in the default setting, "Off".
+
+![image.png](/docs/images/SMARTonFHIR/image-ef9fa38f-a49c-47cf-965f-8a506b0664c1.png)
+
+# Enable CORS
+
+Select "Enable Access-Control-Allow-Credentials" and add your application url. You can use the wildcard "*" instead of a specific url, it is not recommended for security reasons.
```

Comment:
Maybe leave out the wildcard piece since we don't recommend it.

Reply:
If we leave out the wildcard note, people may just use it. So I think it's better to keep it.

Reply:
Sounds good, could we reword it to "While you can use the wildcard "*" instead of a specific URL, we do not recommend this for security reasons".

Reply:
Yes that looks good.